src/components/DateTime.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState, useEffect } from "react";
+
+function DateTime() {
+  const [dayName, setDayName] = useState("Monday");
+  const [date, setDate] = useState("01/01/21");
+  const [time, setTime] = useState("12:00AM");
+
+  const getDateInfo = () => {
+    const days = [
+      "Sunday",
+      "Monday",
+      "Tuesday",
+      "Wednesday",
+      "Thursday",
+      "Friday",
+      "Saturday",
+    ];
```

Comment:
days constant can be extracted out of the function DateTime() as a field so it doesn't get created after every re-render